frontend/src/component/common/PermissionButton/PermissionButton.tsx

```diff
@@ -55,7 +55,16 @@ const ProjectEnvironmentPermissionButton: React.FC<IProjectPermissionButtonProps
             props.environmentId,
         );
 
-        return <BasePermissionButton {...props} access={access} ref={ref} />;
+        return (
+            <BasePermissionButton
+                {...props}
+                access={access}
+                ref={ref}
+                sx={{
+                    backgroundColor: 'red',
```

Comment:
I assume that this is only present for debugging reasons and we don't want this to be merged.

Reply:
thanks! This one slipped :red_square: